XML/TestCases/FunctionalTests-SRIOV.xml

```diff
@@ -176,6 +176,7 @@
     <Tags>network</Tags>
     <Priority>1</Priority>
     <SetupConfig>
+      <Networking>SRIOV</Networking>
```

Comment:
Can you check other lines with this option in the same file? It looks reformating missed the same Networking tag in a few other test cases. @abhimarathe  FYI, this is why we did not see SR-IOV interface in VHM debugging.

Reply:
@harz566  Can you confirm once if all the Extra nics that were added had SRIOV enabled? coz before this tag added this testcase was executing successfully but all the NICs added were Synthetic and not SRIOV.

Reply:
Seems HyperV platform doesn't need that setting, and Azure platform may not need that either, as SRIOV can be enabled from VM itself.  Just come upon this SRIOV is missing in this test case, because we are fixing another problems.

Reply:
> @harz566 Can you confirm once if all the Extra nics that were added had SRIOV enabled? because before this tag was added, this testcase was executing successfully but all the NICs added were Synthetic and not SRIOV. @juhlee-microsoft Correct me if I'm wrong.

Added extra nics with SRIOV enabled now.

Reply:
This is not what I asked. Just shared my comment with @abhimarathe for his next work items. I can close the comment that was originally for his information only.